test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -629,89 +606,28 @@ private class TrackDispose
         public class ConnectionWrapper : IDisposable
         {
             private static int _id;
-            private readonly TestChannel<Message> _input;
-
+            
             public Connection Connection { get; }
 
-            public ChannelConnection<Message> Transport { get; }
-
-            public Task ApplicationStartedReading => _input.ReadingStarted;
+            public IChannelConnection<Message> Application { get; }
 
             public ConnectionWrapper(string format = "json")
             {
                 var transportToApplication = Channel.CreateUnbounded<Message>();
                 var applicationToTransport = Channel.CreateUnbounded<Message>();
 
-                _input = new TestChannel<Message>(transportToApplication);
-
-                Transport = new ChannelConnection<Message>(_input, applicationToTransport);
+                Application = ChannelConnection.Create(input: applicationToTransport, output: transportToApplication);
+                var transport = ChannelConnection.Create(input: transportToApplication, output: applicationToTransport);
 
-                Connection = new Connection(Guid.NewGuid().ToString(), Transport);
+                Connection = new Connection(Guid.NewGuid().ToString(), transport);
                 Connection.Metadata["formatType"] = format;
-                Connection.User = new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.Name, Interlocked.Increment(ref _id).ToString()) }));
+                Connection.User = new ClaimsPrincipal(new GenericIdentity(Interlocked.Increment(ref _id).ToString()));
```

Comment:
Unrelated to the Channel changes, it was from some local testing with Redis and needing the User to be authenticated